### ISSUE TITLE
Draft of explanation of Advanced search: Morphology

### ISSUE THREAD

**Issue by Labrus** [constructicon/russian-data#11]
## 2. Advanced search: Morphology

In advanced search, it is possible to search for constructions that contain slots and anchors restricted by certain morphological categories. In the search window, the morphological tags are listed in alphabetical order. The table below offers an overview of these tags as a system. Here, we list the tags in full, and provide corresponding abbreviations that appear in the search window in parenthesis ().

|Case |Gender|Number|Person|
|-----------------|-----------------|----------------|-------------------|
|-Nominative (Nom)|- Masculine (M)|- Singular (Sg)|- First person (1)|
|-Genitive (Gen)|- Feminine (F)|- Plural (Pl)|- Second person (2)|
|-Partitive genitive (Gen2)|- Neuter (N)|                |- Third person (3)|
|-Dative (Dat)|                 |                |                   |
|-Accusative (Acc)|                 |                |                   |
|-Instrumental (Ins)|                 |                |                   |
|-Locative (Loc)|                 |                |                   |

|Aspect|Mood|Tense|
|----------|--------|-------|
|- Perfective (Pfv)|- Imperative (Imp)|- Past (Pst)|
|- Imperfective (Ipfv)|- Subjunctive (Sbjv)|- Present (Prs)| 
|               |              |- Future (Fut)|

|Lexical parts of speech|Lexical parts of speech|Functional parts of speech|
|---------------------|---------------------|---------------------|
|- Noun not forming an NP (Noun)|- Verb:         |- Preposition (Prep)            |
|- Adjective (Adj)|** Auxiliary (Aux)     |- Conjunction:|
|- Numeral: |** Bare (Bare)|** Coordinating conjunction (Cconj)|
|** Collective numeral (NumColl)|** Copula (Cop)  |** Subordinating conjunction (Sconj)|
|** Cardinal numeral (NumCrd)|** Converb (Cnv)       |- Interjection (Intj)           |
|** Ordinal numeral (NumOrd)|** Infinitive (Inf)    |**Syntactic units** |
|- Pronoun:|** Active participle (PartAct) |- Clause (Cl)|
|** Demonstrative pronoun (PronDem)|** Passive participle (PartPass) |- Noun phrase (NP)|
|** Interrogative/relative pronoun (PronInt)|** Verb not forming a VP (Verb)|- Prepositional phrase (PP)|
|** Personal pronoun (PronPers)|- Adverb (Adv)|- Verb phrase (VP)|
|** Possessive pronoun (PronPoss) |- Predicative (Pred)|- Any phrasal unit (XP)|

|Other|
|---------------|
|- Comparative (Cmp)|
|- Short form of an adjective or a participle (Short)|

**Comment by Labrus** [comment]
I am not skilled enough in Markdown to fix the equal width for columns, to make it nice-looking. I used ** to mark sublists, because I do not forma this properly. For example, Numeral is just a heading, and the Collective numeral (NumColl), Cardinal numeral (NumCrd), Ordinal numeral (NumOrd) are the types of Numeral and are tags that appear in the search window. Also, if you think that this very ugly, maybe we can make this visualisation in PowerPoint and insert a snapshot in the html?

**Comment by Labrus** [comment]
It is also possible to merge Table 1 and Table 2 into a single table like this:

|Case |Gender|Number|Person|
|-----------------|-----------------|----------------|-------------------|
|-Nominative (Nom)|- Masculine (M)|- Singular (Sg)|- First person (1)|
|-Genitive (Gen)|- Feminine (F)|- Plural (Pl)|- Second person (2)|
|-Partitive genitive (Gen2)|- Neuter (N)|                |- Third person (3)|
|-Dative (Dat)|    **Aspect**   |   **Mood**   |   **Tense**   |
|-Accusative (Acc)|- Perfective (Pfv)|- Imperative (Imp)|- Past (Pst)|
|-Instrumental (Ins)|- Imperfective (Ipfv)|- Subjunctive (Sbjv)|- Present (Prs)|
|-Locative (Loc)|               |              |- Future (Fut)|

**Comment by Labrus** [comment]
This is how it could look like (drawn now in Ppt):

<img width="1032" alt="Screen Shot 2021-03-15 at 17 09 41" src="https://user-images.githubusercontent.com/63615172/111184995-d25dcb80-85b1-11eb-9580-b9803c792838.png">
or
<img width="1035" alt="Screen Shot 2021-03-15 at 17 11 51" src="https://user-images.githubusercontent.com/63615172/111185010-d5f15280-85b1-11eb-8a45-f09b106294dc.png">
or
<img width="1028" alt="Screen Shot 2021-03-15 at 17 12 44" src="https://user-images.githubusercontent.com/63615172/111185046-ddb0f700-85b1-11eb-9a18-a47fab5f382d.png">

**Comment by Labrus** [comment]
And the table for grammatical classes (parts of speech & syntactic units would look like this). The terms with dots are the tags, others are headings. Too advanced to format this in -Markdown for me.

<img width="1015" alt="Screen Shot 2021-03-15 at 17 41 22" src="https://user-images.githubusercontent.com/63615172/111188799-b9571980-85b5-11eb-9eb3-6e164c26afe5.png">

**Comment by olesar** [comment]
> This is how it could look like (drawn now in Ppt):
> 
> <img alt="Screen Shot 2021-03-15 at 17 09 41" width="1032" src="https://user-images.githubusercontent.com/63615172/111184995-d25dcb80-85b1-11eb-9580-b9803c792838.png">
> 
> or
> <img alt="Screen Shot 2021-03-15 at 17 11 51" width="1035" src="https://user-images.githubusercontent.com/63615172/111185010-d5f15280-85b1-11eb-8a45-f09b106294dc.png">
> 
> or
> <img alt="Screen Shot 2021-03-15 at 17 12 44" width="1028" src="https://user-images.githubusercontent.com/63615172/111185046-ddb0f700-85b1-11eb-9a18-a47fab5f382d.png">

I like the second option, visually.

**Comment by olesar** [comment]
> And the table for grammatical classes (parts of speech & syntactic units would look like this). The terms with dots are the tags, others are headings. Too advanced to format this in -Markdown for me.
> 
> <img alt="Screen Shot 2021-03-15 at 17 41 22" width="1015" src="https://user-images.githubusercontent.com/63615172/111188799-b9571980-85b5-11eb-9eb3-6e164c26afe5.png">

Looks fine. Predicative can go just after the Verb section since its functions are closer to the Verb

**Comment by bast** [comment]
Just want to add that if Markdown is not flexible enough, it does not have to be Markdown. It can also be HTML styled with CSS. And my understanding is that this is not something that needs to be fixed in the data repository but rather in the app? If yes, then it should be moved to https://github.com/constructicon/russian/issues?

**Comment by Labrus** [comment]
Yes, this is true. I have put this here because I thought I should get it done myself, but apparently I cannot unless we insert the images prepared in pdf.
